proposals/description/tip_121a.ts

```diff
@@ -0,0 +1,179 @@
+import { ethers } from 'ethers';
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const tip_121a: TemplatedProposalDescription = {
+  title: 'TIP-121a: PCV Consolidation, part 1',
+  commands: [
+    // 1. WETH -> DAI auction
+    {
+      target: 'ethToDaiLBPSwapper',
+      values: '0',
+      method: 'setReceivingAddress(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Set the target of the WETH>DAI auction'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: (addresses) => [
+        addresses.ethToDaiLBPSwapper, // to
+        ethers.utils.parseEther('3000000') // amount, 3M
+      ],
+      description: 'Move 3M DAI to the WETH>DAI swapper'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.wethHoldingPCVDeposit, // pcvDeposit
+        addresses.ethToDaiLBPSwapper, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all WETH to the WETH>DAI swapper'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawWrapETH(address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.rariPool146EthPCVDeposit, // from
+        addresses.ethToDaiLBPSwapper, // to
+        '71347946884944201518' // 71.34 ETH in Fuse pool 146
+      ],
+      description: 'Move all ETH in Fuse pool 146 to the WETH>DAI swapper'
```

Comment:
Tribe DAO is the only depositor there and it would require a new contract to read balance at runtime, I think that's good enough, to be discussed at dev meeting

Reply:
ratio controller can't do it?